code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1318,3 +1318,20 @@
 	to_chat(src, span_danger("You shove [target.name] into [name]!"))
 	log_combat(src, target, "shoved", "into [name]")
 	return COMSIG_CARBON_SHOVE_HANDLED
+
+
+// Checks to see how many hands this person has to sign with.
+/mob/living/carbon/proc/check_signables_state()
+	var/obj/item/bodypart/left_arm = src.get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/right_arm = src.get_bodypart(BODY_ZONE_R_ARM)
```

Comment:
```suggestion
	var/obj/item/bodypart/left_arm = get_bodypart(BODY_ZONE_L_ARM)
	var/obj/item/bodypart/right_arm = get_bodypart(BODY_ZONE_R_ARM)
```

Reply:
Right, I'll fix that & also the handcuffed check too